cli/lib/exec/spawn.js

```diff
@@ -113,7 +113,7 @@ module.exports = {
         const electronArgs = _.clone(args)
         const node11WindowsFix = isPlatform('win32')
 
-        if (verify.needsSandbox()) {
+        if (!options.dev && verify.needsSandbox()) {
```

Comment:
why is this necessary? in dev mode we want to be as closely aligned to how it works in production as possible.

Reply:
i'm not sure it's necessary, but it works because in `dev` mode the cli launches `node`, not the `Cypress` binary directly

Reply:
if you call Node with this option it blows up
```
$ node --no-sandbox
node: bad option: --no-sandbox
~/git/cypress-test-tiny on master*
$ echo $?
9
```